include/variables.h

```diff
@@ -265,6 +265,8 @@ extern PlayerData gPlayerData;
 
 #if VERSION_PAL
 extern s32 gCurrentLanguage;
+#else
+#define gCurrentLanguage 0
```

Comment:
Same here
```suggestion
// Only English is supported
#define gCurrentLanguage LANGUAGE_EN
```

Reply:
this is also 0 rather than english for the reason above